template/frameworks/vuetify/layouts/default.vue

```diff
@@ -83,7 +83,7 @@
       :fixed="fixed"
       app
     >
-      <span>&copy; 2019</span>
+      <span>&copy; {{ copyrightYear }}</span>
```

Comment:
Why not just `{{ new Date().getFullYear() }}` instead of extra variable in `data()`